docs/changelog.txt

```diff
@@ -35,6 +35,14 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ======== IMPORTANT: rename this, and add a new "future" section, BEFORE ========
 ======== making a new DFHack release!                                   ========
 ================================================================================
+# Future
+
+## Fixes
+- `stocks`: fixed display of titles by using the new ``Items::`getTitle`` operation before the original one, issue 1262
```

Comment:
Should update this - and also, not sure what the backtick after the `::` is doing here

Reply:
```suggestion
- `stocks`: fixed display of titles by using the new ``Items::getBookTitle`` operation before the original one
```

(we historically haven't put issues in here, and the changelog script probably won't handle links well... I suppose I could address that)